AnkiDroid/src/main/assets/scripts/card.js

```diff
@@ -228,3 +228,25 @@ var onPageFinished = function() {
         .then(() => card.classList.add("mathjax-rendered"))
         .then(_runHook(onShownHook))
 }
+
+
+/**
+* Add function 2 hook to function 1
```

Comment:
I'd like some comment for developers so they know what to send in when adding a hook and how it will be called - I think this conforms with the implementation?

```suggestion
* Add function 2 hook to function 1.
* Function 2 should be `(arg: Object) => void`;  `arg` will be an Object returned from `JSON.parse`
```

Reply:
It is nice.